### ISSUE TITLE
[Beta] ExpandCurrencyList error on login

### ISSUE THREAD

**Issue by amiyuy** [arkayenro/arkinventory#1369]
Beta: 9.0.2.35938

Ark Inventory: v3.09.37 and v3.09.39 (Alpha 5)

```
Message: Interface\AddOns\ArkInventory\ArkInventory.lua:12469: ...dOns\ArkInventory\ArkInventoryCollectionCurrency.lua:100: attempt to call global 'ExpandCurrencyList' (a nil value)
Time: Tue Sep 22 15:30:51 2020
Count: 2
Stack: Interface\AddOns\ArkInventory\ArkInventory.lua:12469: ...dOns\ArkInventory\ArkInventoryCollectionCurrency.lua:100: attempt to call global 'ExpandCurrencyList' (a nil value)
[string "=[C]"]: ?
[string "@Interface\AddOns\ArkInventory\ArkInventory.lua"]:12469: in function `ThreadResume'
[string "*:OnUpdate"]:1: in function <[string "*:OnUpdate"]:1>

Locals: (*temporary) = "...dOns\ArkInventory\ArkInventoryCollectionCurrency.lua:100: attempt to call global 'ExpandCurrencyList' (a nil value)"
```

**Comment by arkayenro** [comment]
looks like they removed that function.  will be fixed in the next alpha/release

**Comment by arkayenro** [comment]
fixed in 3.09.39
